warehouse-management-service/README.md

```diff
@@ -0,0 +1,23 @@
+# Warehouse Management System
+
+Refer [project document](https://docs.google.com/document/d/19Gtx_CG7ebD9HDvTI_KJS9Ge-hfUYPUMx85ZurVgLXs/edit?usp=sharing)
+
+This application is a web service, with reasonable domain complexity. \
+The project is intended to demonstrate/achieve:
+
+1. Relational database modelling
+2. Project management
+3. CI/CD using GitHub Actions and a shell script
+
```

Comment:
Although this is the goal in your onboarding, the README should tell how the application works IMO. You can add these as side goals or describe them separately.

Reply:
Makes sense, I'll remove this from README.